model/src/test/java/org/alfresco/transform/messages/TransformStackTest.java

```diff
@@ -36,15 +38,13 @@
 import java.util.Map;
 import java.util.UUID;
 
-import static org.alfresco.transform.router.TransformStack.OPTIONS_LEVEL;
-import static org.alfresco.transform.router.TransformStack.SEPARATOR;
-import static org.alfresco.transform.router.TransformStack.TOP_STACK_LEVEL;
-import static org.alfresco.transform.router.TransformStack.getInitialSourceReference;
+import static org.alfresco.transform.messages.TransformStack.OPTIONS_LEVEL;
+import static org.alfresco.transform.messages.TransformStack.SEPARATOR;
+import static org.alfresco.transform.messages.TransformStack.TOP_STACK_LEVEL;
+import static org.alfresco.transform.messages.TransformStack.getInitialSourceReference;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
 import static org.mockito.Mockito.doReturn;
```

Comment:
Why remove theses static imports? It has just added more to the diff.

Reply:
If we are doing a clean up, unused imports are one of them.